### PR TITLE
LP-2461 - Integrate Application Hub Page (Adding design fixes)

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
- Change model method to cater changes in progress percentage sent
- Update tests associated to changed model

Previous [PR](https://github.com/OmnipreneurshipAcademy/edx-platform/pull/59) for LP-2461
Previous [PR for design](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/49) of LP-2461

[PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/57) for design fix of LP-2461

### CODE REVIEW

**Review by asadali145 — APPROVED** [review]
LGTM

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/633/workflows/workflows_id/jobs/633/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (100%)
-------------
Total:   1 line
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/applications/models.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def percentage_of_objectives_completed(self):
         """
         number_of_objectives_completed = sum([self.is_written_application_completed,
                                               self.is_prerequisite_courses_passed])
-        return '{0:.0%}'.format(number_of_objectives_completed / self.TOTAL_APPLICATION_OBJECTIVES)
+        return number_of_objectives_completed / self.TOTAL_APPLICATION_OBJECTIVES
```
Comment (ahmed-faraz46) [inline openedx/adg/lms/applications/models.py:69]: We should multiply it with 100. That would align with the name of the method. But it is not just to align with the name of the method.

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/636/workflows/workflows_id/jobs/636/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/test/test_models.py (100%)
-------------
Total:   2 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**